index.js

```diff
@@ -4030,6 +4037,39 @@
   S.splitOn =
   def('splitOn', {}, [$.String, $.String, $.Array($.String)], splitOn);
 
+  //# sortBy :: (a -> a -> Ordering) -> Array a -> Array a
+  //.
+  //. Returns a copy of the list, sorted according to the comparator
+  //. function, which should accept two values at a time and return a
+  //. negative number if the first value is smaller, a positive number if
+  //. it's larger, and zero if they are equal. Please note that this is a
+  //. **copy** of the list. It does not modify the original.
```

Comment:
Let's make some changes to this description:

  - replace occurrences of "list" with "array";
  - replace mentions of negative numbers, zero, and positive numbers; and
  - remove mentions of copying (it should go without saying that Sanctuary will not mutate arguments).